modules/recon/domains-hosts/certificate_transparency.py

```diff
@@ -14,7 +14,7 @@ class Module(BaseModule):
     def module_run(self, domains):
         for domain in domains:
             self.heading(domain, level=0)
-            resp = self.request(f"https://crt.sh/?q=%25.{domain}&output=json")
+            resp = self.request(f"https://crt.sh/?q=%25.{domain}&output=json", timeout=20)
```

Comment:
Users are supposed to manage this with the global timeout option. That's why it exists. Is that an acceptable solution rather than changing the module?

Reply:
There is a global timeout option????? Guess I missed that. Let me test with that real quick but I would assume yep that would solve this issue.

Reply:
Yep that works. Closing this